### ISSUE TITLE
Reconcile MMDB DTXSIDs not registered in Factotum

### ISSUE THREAD

**Issue by Sakshi-Handa** [HumanExposure/factotum#2211]
Recently, when we uploaded data for the MMDB Summary Data tables, we were unable to add 5.6k records associated with 892 unique DTXSIDs that are not present in Factotum. Since we added entries to the table based on the DTXSID as the foreign key, when a certain SID was not registered in Factotum, the MMDB record associated with that SID was unable to transfer.
 
The attached .csv shows the list of missing DTXSIDs, and additional curated chemistry info associated with the missing MMDB records (including chemical name, cas, RID, etc)

Questions:
What can be done to account for this difference in the chemical space? 
Would we be able to create DSS TOX lookup values for the missing chemical SIDs? 

Definition of Done:
Requires discussion.

**Comment by Sakshi-Handa** [comment]
[missing_mmdb_dtxsid_20220222.csv](https://github.com/HumanExposure/factotum/files/8201272/missing_mmdb_dtxsid_20220222.csv)

**Comment by mwfrost** [comment]
Yes, the fix would be to add these records to `dashboard_dsstoxlookup`

**Comment by Sakshi-Handa** [comment]
A tip for completing this ticket:
On the /chemical_curation/ page, we have the functionality to 'upload curated chemicals csv file'. 
That is the process of how we upload curated chemical records from ChemReg and assign them with the corresponding raw chemical ids in Factotum. 
When we upload a NEW chemical (meaning a chemical we haven't seen in Factotum before), part of that process is to create a new record to the dashboard_dsstoxlookup table with the sid, true_chemical_name, true_cas. 
So the ability to add new DSSTOX Lookup values already exists within this tool, that can probably be referenced/replicated in the new functionality of this ticket.
